### ISSUE TITLE
massive multi body assembly

### ISSUE THREAD

**Issue by CPME** [bernhard-42/jupyter-cadquery#6]
I cannot view an entire assembly of 900 elements (see attached image). 400 element assembly is fully viewable. How many elements show up seems indeterminate, as it is different every time I run the script
![chainmail-viewer-bug](https://user-images.githubusercontent.com/17184913/67629640-f0e4ea80-f835-11e9-8084-ea786b1b15cd.png)
. Interpreter does not throw error. Here is the code that generates chainmail.

`import cadquery as cq
from jupyter_cadquery.cadquery import (Assembly, Part, Edges, Faces, Vertices, show)
from jupyter_cadquery import set_sidecar

set_sidecar("CadQuery")

ring_od = 5
ring_id = 4
ring_t = 1
clearance = 1
ring_offset = ring_od+clearance
n_x = 30
n_y = 30
x_hat = (1, 0, 0)
y_hat = (0, 1, 0)
z_hat = (0, 0, 1)

ring = cq.Workplane('XY').circle(ring_od).circle(ring_id).extrude(ring_t)
ringx = ring.rotate((0, 0, 0), (1, 0, 0), 90).translate((ring_offset, 0, 0))
ringy = ring.rotate((0, 0, 0), (0, 1, 0), 90).translate((0, ring_offset, 0))
ringy2 = ring.rotate((0, 0, 0), (0, 1, 0), 90).translate((0, 2*ring_offset, 0))
unit_cell = ring.union(ringy).union(ringx)
cells_x = [unit_cell.translate((2 * x * ring_offset, 0, 0)) for x in range(n_x)]
cells_y = [[i.translate((0, 2 * y * ring_offset, 0)) for i in cells_x] for y in range(n_y)]
flattened_list = [y for x in cells_y for y in x]

def assemble(parts=[]):
    return Assembly([Part(part, "red box",   "#d7191c", show_edges=False) for part in parts],"test")

ring_assembly = assemble(flattened_list)

show(ring_assembly, axes=True)
`

**Comment by CPME** [comment]
Ran it again with no modifications, and it generated the entire model.
![chainmail-900-elements](https://user-images.githubusercontent.com/17184913/67630388-f9dcb880-f843-11e9-9427-4b21eb337283.png) Any ideas?

**Comment by bernhard-42** [comment]
@CPME Firstly, thanks for giving jupyter-cadquerya try!

I would have expected that memory is an issue, so I tried your example on an (already swapping) 8GB mac laptop and a bit 128GB linux machine. In both cases the model rendered fine.

I then moved on to 50x50 rings on the linux machine and it stopped in the middle. 

![image](https://user-images.githubusercontent.com/954408/67632690-eca4e580-f8a6-11e9-9065-b6392608450c.png)

However I received the error from jupyterlab

![image](https://user-images.githubusercontent.com/954408/67632665-a0f23c00-f8a6-11e9-9b3b-5d7b3c704ece.png)

Did you see something like this? Would changing iopub rates on your machine help?
